code/modules/mob/living/carbon/human/human.dm

```diff
@@ -867,8 +867,12 @@
 			return
 		//If you dragged them to you and you're aggressively grabbing try to fireman carry them
 		else if(user != target)
-			fireman_carry(target)
-			return
+			if(user.a_intent == INTENT_GRAB || INTENT_DISARM || INTENT_HARM)
+				to_chat(user, "<span class='warning'>You need to be in help intent to do that!</span>")
+				return

```

Comment:
The PR was just fine as is before these last commits (maybe the intent should have been grab, though). Now it'll just annoy whoever attempts to strip or fireman carry a human mob outside of help intent with an obnoxious feedback message.

Reply:
yeah idk catgirl lineman wanted a message but better solution would be to change stripping and firemancarry to combat mode and not combat mode, combat mode for stripping (no more crashing your game while trying to click someone)

Reply:
i know you fixed this but i should also note that "user.a_intent == INTENT_GRAB || INTENT_DISARM || INTENT_HARM" will always evaluate to true, since it's checking if INTENT_DISARM is truthy rather than checking if a_intent is INTENT_DISARM